### ISSUE TITLE
FEATURE - Support all Dynamic Type sizes throughout the app

### ISSUE THREAD

**Issue by SuzGupta** [mikaelacaron/brain-marks#136]
**Is your feature request related to a problem? Please describe.**
Anyone with aging eyes (hello!) or vision issues relies on Dynamic Type. All apps should support this.

**Describe the solution you'd like**
Full support for all sizes of Dynamic Type throughout the app

**Describe alternatives you've considered**
Partial support for Dynamic Type (if that's a thing); just not supporting Dynamic Type and trying for a fairly large, readable hard-coded font or set of fonts... we know that supporting Dynamic Type is a best practice though and should be relatively easy.

**Additional context**
Apple doc on Dynamic Type: https://developer.apple.com/documentation/uikit/uifont/scaling_fonts_automatically

**Comment by Miablo** [comment]
May I be assigned this issue as well? It looks fairly straight forward... unless i'm missing something.

**Comment by mikaelacaron** [comment]
@Miablo same comment as [here](https://github.com/mikaelacaron/brain-marks/issues/139#issuecomment-1264548644) right now, I'd like to limit it to one issue per person. But if you finish the issue you're assigned, more than welcome to grab new ones!

**Comment by aopoka** [comment]
Hey! Can I be assigned to this issue? It would be my first one :)

**Comment by mikaelacaron** [comment]
@aopoka Go for it! I think it's mainly set up to support dynamic type inherently, but there may be some UI changes here and there when at very large or small dynamic type sizes that should be verified for the best experience

**Comment by mikaelacaron** [comment]
@aopoka how's it going working on this issue? Do you have any questions? Do you think you'll complete it by the end of Hacktoberfest?

**Comment by aopoka** [comment]
Hi @mikaelacaron, I'm having trouble building the project, I pulled the latest version from the `dev` branch but I get a lot of errors connected to the `Logger` and string interpolation. Are you facing similar issues?

**Comment by mikaelacaron** [comment]
@aopoka  I haven't! Have you read the readme and contributing documents?
